source/extensions/extensions_build_config.bzl

```diff
@@ -3,95 +3,87 @@ EXTENSIONS = {
-    "envoy.access_loggers.http_grpc":                   "//source/extensions/access_loggers/http_grpc:config",
+    "envoy.access_loggers.file": "//source/extensions/access_loggers/file:config",
+    "envoy.access_loggers.http_grpc": "//source/extensions/access_loggers/http_grpc:config",
 
     #
     # gRPC Credentials Plugins
     #
-
-    "envoy.grpc_credentials.file_based_metadata":      "//source/extensions/grpc_credentials/file_based_metadata:config",
+    "envoy.grpc_credentials.file_based_metadata": "//source/extensions/grpc_credentials/file_based_metadata:config",
 
     #
     # Health checkers
     #
-
-    "envoy.health_checkers.redis":                      "//source/extensions/health_checkers/redis:config",
+    "envoy.health_checkers.redis": "//source/extensions/health_checkers/redis:config",
 
     #
     # HTTP filters
     #
-
-    "envoy.filters.http.buffer":                        "//source/extensions/filters/http/buffer:config",
-    "envoy.filters.http.cors":                          "//source/extensions/filters/http/cors:config",
-    "envoy.filters.http.dynamo":                        "//source/extensions/filters/http/dynamo:config",
-    "envoy.filters.http.ext_authz":                     "//source/extensions/filters/http/ext_authz:config",
-    "envoy.filters.http.fault":                         "//source/extensions/filters/http/fault:config",
-    "envoy.filters.http.grpc_http1_bridge":             "//source/extensions/filters/http/grpc_http1_bridge:config",
-    "envoy.filters.http.grpc_json_transcoder":          "//source/extensions/filters/http/grpc_json_transcoder:config",
-    "envoy.filters.http.grpc_web":                      "//source/extensions/filters/http/grpc_web:config",
-    "envoy.filters.http.gzip":                          "//source/extensions/filters/http/gzip:config",
-    "envoy.filters.http.health_check":                  "//source/extensions/filters/http/health_check:config",
-    "envoy.filters.http.header_to_metadata":            "//source/extensions/filters/http/header_to_metadata:config",
-    "envoy.filters.http.ip_tagging":                    "//source/extensions/filters/http/ip_tagging:config",
-    "envoy.filters.http.jwt_authn":                     "//source/extensions/filters/http/jwt_authn:config",
-    "envoy.filters.http.lua":                           "//source/extensions/filters/http/lua:config",
-    "envoy.filters.http.ratelimit":                     "//source/extensions/filters/http/ratelimit:config",
-    "envoy.filters.http.rbac":                          "//source/extensions/filters/http/rbac:config",
-    "envoy.filters.http.router":                        "//source/extensions/filters/http/router:config",
-    "envoy.filters.http.squash":                        "//source/extensions/filters/http/squash:config",
+    "envoy.filters.http.buffer": "//source/extensions/filters/http/buffer:config",
```

Comment:
Can we avoid having Buildifier apply to this situation? It seems there are advantages to readability for these configuration files in having block spacing.

Reply:
`# buildifier: leave-alone` is ignored for dictionaries, to the only option is to completely skip `buildifier` for this file (which is effectively the same thing, since this dictionary is the only thing in that file).

Having said that, both `buildifier` and `clang-format` are sacrificing readability for consistency on a regular basis, and I'm not sure if this is the worse offender and/or if we want to be in business of adding one-off exceptions.

I don't care too much, so I can exclude this file if you want. Your call.

Reply:
Prefer we ignore in this case, since this is human read/write configuration, vs. code per se.

Reply:
Added to excluded files and reverted to previous version (sans trailing whitespace).